app/config/collections.php

```diff
@@ -3381,6 +3381,99 @@
             ],
         ]
     ],
+
+    'variables' => [
+        '$collection' => Database::METADATA,
+        '$id' => 'variables',
+        'name' => 'variables',
+        'attributes' => [
+            [
+                '$id' => 'functionInternalId',
+                'type' => Database::VAR_STRING,
+                'format' => '',
+                'size' => Database::LENGTH_KEY,
+                'signed' => true,
+                'required' => false,
+                'default' => null,
+                'array' => false,
+                'filters' => [],
+            ],
+            [
+                '$id' => 'functionId',
+                'type' => Database::VAR_STRING,
+                'format' => '',
+                'size' => Database::LENGTH_KEY,
+                'signed' => true,
+                'required' => false,
+                'default' => null,
+                'array' => false,
+                'filters' => [],
+            ],
+            [
+                '$id' => 'key',
+                'type' => Database::VAR_STRING,
+                'format' => '',
+                'size' => Database::LENGTH_KEY,
+                'signed' => true,
+                'required' => false,
+                'default' => null,
+                'array' => false,
+                'filters' => [],
+            ],
+            [
+                '$id' => 'value',
+                'type' => Database::VAR_STRING,
+                'format' => '',
+                'size' => 16384,
```

Comment:
Size of `vars` does not really matter, it will always be `null`. I set that one to 16k since all other subqueries uses 16k. not sure why.

Good point tho, previously value was 8k, not 16k. So I will decrease this to `8192`.